tools/azsdk-cli/Azure.Sdk.Tools.Cli.Evaluations/Scenarios/AzureRestApiSpecs/Evaluate_GetModifiedTypespecProjects.cs

```diff
@@ -0,0 +1,42 @@
+using Azure.Sdk.Tools.Cli.Evaluations.Evaluators;
+using Azure.Sdk.Tools.Cli.Evaluations.Helpers;
+using Azure.Sdk.Tools.Cli.Evaluations.Models;
+using Microsoft.Extensions.AI.Evaluation;
+using NUnit.Framework;
+
+namespace Azure.Sdk.Tools.Cli.Evaluations.Scenarios
+{
+    public partial class Scenario
+    {
+        [Test]
+        [Category(RepositoryCategories.AzureRestApiSpecs)]
+        public async Task Evaluate_GetModifiedTypespecProjects()
+        {
+            const string prompt = "List the TypeSpec projects modified in my repo. My setup has already been verified, do not run azsdk_verify_setup. Path to root: C:\\azure-rest-api-specs. Compare against main.";
```

Comment:
Seems like we could use a way to always skip verify setup so these scenario prompts don't have to duplicate this instruction